GameServerLib/API/ApiFunctionManager.cs

```diff
@@ -149,6 +149,30 @@ public static void RemoveParticle(Particle p)
             _game.PacketNotifier.NotifyParticleDestroy(p);
         }
 
+        public static Minion AddMinion(IChampion champion, string model, string name, float toX, float toY, int visionRadius = 0)
+        {
+            var m = new Minion(_game, champion, toX, toY, model, name, visionRadius);
+            _game.ObjectManager.AddObject(m);
+            m.SetVisibleByTeam(champion.Team, true);
+            _game.PacketNotifier.NotifyMinionSpawned(m, champion.Team);
+            return m;
+        }
+
+        public static Minion AddMinionTarget(IChampion champion, string model, string name, ITarget target, int visionRadius = 0)
+        {
+            var m = new Minion(_game, champion, target.X, target.Y, model, name, visionRadius);
+            _game.ObjectManager.AddObject(m);
+            m.SetVisibleByTeam(champion.Team, true);
+            _game.PacketNotifier.NotifyMinionSpawned(m, champion.Team);
```

Comment:
`AddObject` already send `NotifyMinionSpawned`. (see `OnAdded` function for minion)